docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/BootstrapSample.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Data.DataView;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class BootstrapSample
```

Comment:
>public class BootstrapSample [](start = 4, length = 28)

Is this intended to be a static class? It has no modifier.

Reply:
Let's keep it static.